### ISSUE TITLE
Submenu toggle button has redundant title text

### ISSUE THREAD

**Issue by maccabeelevine** [zurb/foundation-sites#11730]
## What should happen?

The Accordion Menu with a Submenu Toggle enabled generates a button next to the parent item, which opens the submenu.  Presumably that button should be accessibility complaint.  WebAIM WAVE flags an issue "Redundant title text" when "A title attribute value is identical to element text or image alternative text."  Therefore the title attribute of the button shouldn't be identical to element text contained within the button.

## What happens instead?

The generated button has a title attribute "Toggle menu".  It contains a span whose contents is "Toggle menu."  That duplication triggers the WAVE alert "Redundant title text".

## Possible Solution

WebAIM says "In most cases the title attribute can be removed, otherwise modify it to provide advisory, but not redundant information."

## Test Case and/or Steps to Reproduce (for bugs)

Test Case: https://foundation.zurb.com/sites/docs/accordion-menu.html

WAVE reports this issue when I scan the Foundation Documentation page for Accordion Menu.  

http://wave.webaim.org/report#/https://foundation.zurb.com/sites/docs/accordion-menu.html

Under the Submenu Toggle code sample, the button DOM generated for the first parent menu (via Chrome inspector) is

`<button id="isgmsr-acc-menu-link" class="submenu-toggle" aria-controls="3zcsy3-acc-menu" aria-expanded="false" title="Toggle menu">
  <span class="submenu-toggle-text">Toggle menu</span>
</button>`

Note "Toggle menu" repeated.

## Context

I am trying to maximize accessibility compliance of a website that uses accordion menu on mobile screen widths.

Note: I see another accessibility issue reported about this same button, but it's a different issue: https://github.com/zurb/foundation-sites/issues/11335

## Your Environment

- Foundation version(s) used: 6.5.3 (Assuming Foundation's documentation uses the version that it documents.)
- Browser(s) name and version(s): Chrome 73 
- Device, Operating System and version: Windows 10

## Checklist

- [X ] I have read and follow the CONTRIBUTING.md document.
- [X ] There are no other issues similar to this one.
- [X ] The issue title and template are correctly filled.

**Comment by ihscmagill** [comment]
@joeworkman This is a valid issue. I'm not sure why it was closed.
